app/models/score.rb

```diff
@@ -22,4 +24,10 @@ def serialize
   def future_score
     errors.add(:played_at, 'must not be in the future') if played_at > Time.zone.today
   end
+
+  def correct_total_score
+    errors.add(:total_score, 'wrong total score')  
+      if (number_of_scores == 9 && (total_score < 27 || total_score >= 90)) ||
+        (number_of_scores == 18 && (total_score < 90 || total_score >= 180))
```

Comment:
I get an error when running `rails db:seed` related to this code.

<img width="1096" alt="Screenshot 2022-02-17 at 21 37 14" src="https://user-images.githubusercontent.com/66674425/154557558-db60f7e9-3b41-4195-8bfc-bf2f2f6a65eb.png">

I think you are trying to apply an inline if here `statement if condition`, but the ruby parser sees the keyword `if` on the next line and thinks you are going to write a normal if
```ruby
if condition
  statement
end
```

If the condition for the `if` is this long, use a normal if.